pkg/signature/kms/hashivault/client.go

```diff
@@ -48,7 +47,7 @@ var (
 )
 
 const (
-	vaultV1DataPrefix = "vault:v1:"
+	vaultV1DataRegExPrefix = "vault:v[0-9]+:"
```

Comment:
Nit: use `regexp.MustCompile` as above so we don’t recompile the regex every time we call `vaultDecode`